subprojects/docs/src/samples/userguide/mavenMigration/basic/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+plugins {
+    `java-library`
+    checkstyle
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+version = "1.2.1"
+
+val currentBuildNumber by extra("1234")
+
+// tag::repositories[]
+repositories {
+    mavenCentral()
+}
+// end::repositories[]
+
+// tag::compile-dependencies[]
+dependencies {
+    implementation("log4j:log4j:1.2.12")  // <1>
+}
+// end::compile-dependencies[]
+
+// tag::pom-dependencies[]
+dependencies {
+    testImplementation("org.codehaus.groovy:groovy-all:2.5.4")
+}
+// end::pom-dependencies[]
+
+// tag::process-resources[]
+tasks {
+    processResources {
+        expand("version" to version, "buildNumber" to currentBuildNumber)
+    }
+}
+// end::process-resources[]
+
+// tag::checkstyle[]
+checkstyle {
+    config = resources.text.fromFile("checkstyle.xml", "UTF-8")
+    setShowViolations(true)
+    setIgnoreFailures(false)
```

Comment:
:x: should be:
```kotlin
checkstyle {
    config = resources.text.fromFile("checkstyle.xml", "UTF-8")
    isShowViolations = true
    isIgnoreFailures = false
}
```

Reply:
Ah, OK. I thought I was being stung by the issue that requires the setters to be used, but looking back, I see that only applies when the getter and setter methods have different types. This is just a failure to understand the different convention for boolean properties.